doc/high-level-design.md

```diff
@@ -0,0 +1,81 @@
+# High Level Design - dxw towtruck
+
+## What is this document?
+The purpose of this document is to facilitate a more structured and efficient approach to documenting dxw techical projects. 
+
+An HLD aims to describe a digital service or product and then define the service, technology, data and security architecture.
+
+The document is not designed to be a complete Technical Design and should remain high-level, providing a holistic view where multiple components and underpinning services collaborate to deliver a unified solution. It should provide sufficient information for a technical individual who is unfamiliar with the product to understand the scope, the key technology and design decisions, the non-functional requirements and how the solution meets those requirements.
+
+This document should act as a central location to either record information directly or point people in the direction of relevant information.
+
+This document should record important decisions to do with the project.
+
+The template aims to provide prompts and guidelines that help our Tech Leads to consider critical aspects when designing a service or product.
+
+It may not be necessary to complete all sections if they are not relevant to the service or product but they should at least be considered.
+
+
+## Project Overview
+What is the dxw towtruck project?
+
+dxw have over 600 repositories within their own Github account. These need to be maintained and monitored for security updates. New versions of the packages, frameworks, and languages we include within projects are constantly being released. These are not always actioned as it's hard to get a clear picture of the repoositories and emails can be easily missed. It is important that we can easily see what is a priority for updating and what may have been forgotten.
+
+It is proposed that we build a simple, centralised dashboard that allows for a clear overview of the status of our repositories.
+
+### Requirements
+- The service should be accessible to people within dxw
+- The service should pull in all of the dxw owned repositories
```

Comment:
[We have agreed](https://dxw.slack.com/archives/C07KVF495MY/p1725459148483869) that, for now, we should only monitor repos under the dxw Github org. There are some others that we own (EG `UKGovernmentBEIS/beis-report-official-development-assistance`) that are outside the org.